features/date_range_formats.feature

```diff
@@ -51,3 +51,9 @@ Feature: Date range formats
     When I run `calrom 1992-01-02 1992-05-12`
     Then the exit status should be 0
     And the output should match /^\s+January 1992\s+February 1992/
+
+  Scenario: cal-like - -3
+    When I run `calrom -3`
+    Then the exit status should be 0
+    # the output will consider the actual month and year
+    And the output should match /^\s+September 2020\s+October 2020\s+November 2020/
```

Comment:
Also: have you tried executing the cucumber scenarios? (If you don't know the command to do so, it's in `README.md`.) I don't think so, because this expectation fails. The actual output doesn't look like you expect. The expectation needs to be rewritten to pass with the actual output.

Reply:
```ruby
[ 11:13PM ]  [ fsoares@pyhooma:~/Documents/projects/calrom(fs-add-range-date✗) ]
 $ cucumber features/date_range_formats.feature:55
/home/fsoares/.rbenv/versions/2.6.3/lib/ruby/gems/2.6.0/gems/gherkin-2.12.2/lib/gherkin/formatter/filter_formatter.rb:105: warning: constant ::Fixnum is deprecated
Feature: Date range formats
  In order to conveniently select the date range to display
  I want the command to support a few date range entry formats.

  Scenario: cal-like - -3                                                                                                   # features/date_range_formats.feature:55
    When I run `calrom -3`                                                                                                  # aruba-0.14.14/lib/aruba/cucumber/command.rb:6
    Then the exit status should be 0                                                                                        # aruba-0.14.14/lib/aruba/cucumber/command.rb:263
    And the output should match /^\s+January|February|March|April|May|June|July|August|September|October|November|December/ # aruba-0.14.14/lib/aruba/cucumber/command.rb:239

1 scenario (1 passed)
3 steps (3 passed)
0m0.614s
```